### DISCUSSION TITLE
Retro: grouping in two steps

### DISCUSSION THREAD

**Discussion by dargmuesli** [ParabolInc/parabol#9105] [Feedback]
In every Retro my team is getting to the group stage and first wants to wait for everyone to have had the opportunity of reading all cards *before* grouping starts. When the grouping starts cards move confusingly making it incredibly hard to read them all.

Unfortunately, cards are movable right away. What we'd wish for is an additional step ("reading") that allows to click on ready while the cards cannot be moved just yet. Currently, clicking on ready in the grouping category after reading, which team members often do when they have finished reading and have to be told not to do, makes the retro('s facilitator) move on to voting.

Is that feature configurable already by any chance? I've never been the facilitator yet. If not, we'd like to voice our strong interest in that feature! :muscle:

**Comment by Dschoordsch** [comment]
Thank you very much for your feedback!
In our own retros the facilitator reads all cards aloud and only the ones they already read are grouped sometimes, but mostly we wait until all have been read.
I very much appreciate the insight into your workflow. Right now we don't have a setting for this.

**Reply by dargmuesli** [reply]
Yes, we started as well with one person reading all cards aloud. But we noticed that this made it hard for others to find out about the card's contents at their own speed. Right now we're all muted until everyone has raised their hands in the parallel Google Meet we're using to talk when they've finished reading, which adds another level of complexity :grin:
